rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "nightly"
```

Comment:
we prefer to build with the stable version of rust

Reply:
done.